documentation/src/main/docs/faq.adoc

```diff
@@ -117,6 +99,28 @@ To chain asynchronous calls, use `onItem().produceUni` or `onItem().produceCompl
 include::../../../src/test/java/snippets/HowToChainAsyncTest.java[tags=code]
 ----
 
+=== How do I _join_ asynchronous calls?
+
+Joining asynchronous calls, _i.e._ being notified when multiple actions complete is a common pattern.
```

Comment:
Does this need a comma after "complete"?

Reply:
yes